tests/benchmark/linksHandler.ts

```diff
@@ -0,0 +1,40 @@
+import { readFileSync, writeFileSync } from "fs";
+import { join, dirname } from "path";
+import { fileURLToPath } from "url";
+
+export function getCurrentDir(): string {
+	// Get the directory of the current script file and create absolute path
+	const __filename = fileURLToPath(import.meta.url);
+	const __dirname = dirname(__filename);
+	return __dirname;
+}
+
+export function getLinks(
+	n: number = 1000,
+	exportToTSFile: boolean = false,
+): string[] {
+	const __dirname = getCurrentDir();
```

Comment:
Er der en grund til at navnene i denne fil starter med __ ( to underscores )?

Reply:
fordi det matcher de gamle Node navne